src/cpp/session/modules/SessionSpelling.cpp

```diff
@@ -128,6 +146,19 @@ void handleDictionaryRequest(const http::Request& request, http::Response* pResp
    {
       pResponse->setCacheableFile(allLanguagesDir().complete(splat[1]), request);
    }
+
+   /*
+    * \deprecated
+    * Calls to old deprecated dictionary locations for RStudio 1.2 and earlier
+    */
+   if (old_customDictionariesDir().complete(splat[1]).exists())
```

Comment:
Should this be an else if? (otherwise it seems possible to set the file more than once)